src/admin-events/admin-events.json

```diff
@@ -31,7 +31,35 @@
    "FallbackOracleUpdated(address)"
     ],
   "WETHGateway" : [
-    "OwnershipTransferred(address,address)"
+   "OwnershipTransferred(address,address)"
+    ],
+  "AaveGovernanceV2" : [
+    "ExecutorAuthorized(address)",
+    "ExecutorUnauthorized(address)",
+    "GoveranceStrategyChanged(address,address)",
+    "OwnershipTransferred(address,address)",
+    "ProposalCanceled(uint256)",
+    "ProposalCreated(uint256,address,address,address[],uint256[],string[],bytes[],bool[],uint256,uint256,address,bytes32)",
+    "ProposalExecuted(uint256,address)",
+    "ProposalQueued(uint256,uint256,address)",
+    "VoteEmitted(uint256,address,bool,uint256)",
+    "VotingDelayChanged(uint256,address)"
+    ],
+  "ExecutorShort" : [
+    "CancelledAction(bytes32,address,uint256,string,bytes,uint256,bool)",
+    "ExecutedAction(bytes32,address,uint256,string,bytes,uint256,bool,bytes)",
+    "NewAdmin(address)",
+    "NewDelay(uint256)",
+    "NewPendingAdmin(address)"
+    ],
+  "ExecutorLong" : [
+    "CancelledAction(bytes32,address,uint256,string,bytes,uint256,bool)",
+    "ExecutedAction(bytes32,address,uint256,string,bytes,uint256,bool,bytes)",
+    "NewAdmin(address)",
+    "NewDelay(uint256)",
+    "NewPendingAdmin(address)"
+    ],
+  "EcosystemReserve" : [
+    "NewFundsAdmin(address)"
```

Comment:
not your fault, but the indentation in this file is messed up. I know in vim you can fix indentation with `<esc>gg=G`

Reply:
Fixed by using <esc>gg=G in VIM.